### PR TITLE
ember conf day 1 recap

### PR DESCRIPTION
**Pull request by vikram7**
This PR is for the [Greenfield Blog](blog.greenfieldhq.com). It's a recap of EmberConf 2016 Day 1.

### CODE REVIEW

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+layout: post
+title: "EmberConf Recap -- Day 1"
+description: "A short recap of some of the presentations at EmberConf -- Day 1"
+category:
+tags: [ember, ember.js, conference, emberconf]
+author: vikram_ramakrishnan
+---
+{% include JB/setup %}
+
+88.89% of [Greenfield](www.greenfieldhq.com)'s team is attending [EmberConf 2016](http://emberconf.com/schedule.html)!
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md:11]: Sorry not 100%

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+With over 950+ attendees, this EmberConf's largest showing yet. It's an
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: "this is EmberConf's largest showing yet."

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+exciting time to be working in Ember, and this short blog post highlights 
+some of today's presentations:
+
+###Keynote
+This was a really great keynote by [Yehuda Katz](https://twitter.com/wycats) & [Tom Dale](https://twitter.com/tomdale) highlight where Ember is headed, with a particular focus on performance & optimization.
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: keynote by Yehuda & Tom **who** highlight... ???

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+- Ember core team remains but is also dividing up into sub-teams (Ember Data, Ember-CLI, Learning)
+- The Ember core team is really pushing to be more mobile friendly
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: Maybe elaborate a bit more? Sounded at first like maybe the core team wants to be more remote. Instead, I think this is a focus on Ember performance on mobile devices. Especially Android. See https://eviltrout.com/2016/02/25/fixing-android-performance.html

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+- Glimmer2 is phenomenally fast (seeing something in the range of 3-5x improvements in FPS in some examples)
+
+###Service Workers
+[John Kleinschmidt](https://twitter.com/jkleinsc) of [cure.org](https://cure.org/) introduced us to Service Workers in Ember to highlight its benefits.
+
+- Service workers as a reliable replacement for application cache
+- Highlighted benefits of service workers for offline functionality in apps
+- Impressed by how easily service workers can be integrated with Ember-CLi
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: Ember-CLI? ember-cli? EMBER-C-to-the-L-to-the-I?!?!

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+###Cross-Pollinating Communities: We All Win
+[Chris Ball](https://twitter.com/cball_) of [echobind](https://echobind.com/) presented a short history of Ember through multiple vantage points.
+
+- "Good artists good copy, great artists steal"
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: "Good artists copy, great artists steal"

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+- A great historical look at Ember.js from its roots in SproutCore to where we are today
+- Ember has drawn from a number of frameworks (convention over configuration, one-way data binding) and other frameworks have drawn from Ember (Angular’s CLI, React Router)
+
+###<select>ing Good Ember Patterns
+
+[Brenna O'Brien](https://twitter.com/brnnbrn) spent time walking us [through](http://talks.brennaobrien.com/ember-select/selecting-good-ember-patterns.pdf) making good design pattern
+choices in the vein of "Data Down, Actions Up" and Closure Actions to build
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: "Data Down, Actions Up" (DDAU)

So future references make sense

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+a flexible, reusable and modern app.
+
+- Benefits of components with custom HTML & custom data flow
+- Implementing DDAU to help add more dynamism to your Ember app
+- `{{get}}` helper, which lets you [dynamically lookup](http://emberjs.com/api/classes/Ember.Templates.helpers.html#method_get) an object's property &`{{mut}}` helper to implement 1 or 2-way bindings
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: Looks like missing a space &`{{mut}}`

File: _posts/2016-03-28-ember-conf-day-1.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+[Alex Blom](https://twitter.com/AlexBlom) walked us through [ember-cli-cordova](https://github.com/poetic/ember-cli-cordova), a tool to help develop mobile Ember apps.
+
+- Forked over a Platform Service from Ionic to determine what Platform the app is running on
+- Handling touch events with [Hammer.js](http://hammerjs.github.io/). Also, infinite scrolling with [smoke and mirrors](https://github.com/runspired/smoke-and-mirrors) and animations with [liquidfire](https://github.com/ember-animation/liquid-fire) work with Cordova. Another animation library to consider is [velocity.js](https://github.com/julianshapiro/velocity)
+- [Crosswalk Project](https://crosswalk-project.org/) enables some performance optimizations. Managing 'Reflow'
+
+###Warp Speed Memory Management
+
+[Kelly Senna](https://twitter.com/simplysenna) reflected on the importance of memory management
+for a few reasons: (1) User experienced are unequal; (2) Browsers are affected
```
Comment (davekaro) [inline _posts/2016-03-28-ember-conf-day-1.md]: "(1) User experienced are unequal" is this "(1) User experiences are unequal" ?

**Comment by davekaro** [comment]
sweet

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 9 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: :+1:

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 8 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: :+1:

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 7 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: :+1:

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 6 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: Ha :+1:

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 5 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: I was trying to be hip

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 2 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: :+1:

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 3 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: :+1:

File: _posts/2016-03-28-ember-conf-day-1.md (hunk 4 above)
Comment (vikram7) [inline _posts/2016-03-28-ember-conf-day-1.md]: :+1:

**Comment by davekaro** [comment]
:shipit:

**Pull request merged**